src/components/Tabs/Tabs.tsx

```diff
@@ -0,0 +1,43 @@
+import classNames from 'classnames';
+import { Link, useParams } from 'react-router-dom';
+
+const tabs = [
+  { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
+  { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
+  { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
+];
+
+export const Tabs = () => {
+  const { tabId = 0 } = useParams();
+  const selectedTab = tabs.find(tab => tab.id === tabId);
```

Comment:
```suggestion
  const tabContent = tabs
    .find(tab => tab.id === tabId)?.content || 'Please select a tab';
```

You can set content here